daml-assistant/test/DAML/Assistant/Tests.hs

```diff
@@ -398,7 +401,34 @@ testInstall = Tasty.testGroup "DAML.Assistant.Install"
 
 testInstallUnix :: Tasty.TestTree
 testInstallUnix = Tasty.testGroup "unix-specific tests"
-    [ Tasty.testCase "reject an absolute symlink in a tarball" $ do
+    [ Tasty.testCase "initial install a tarball from symlink" $ do
+              withSystemTempDirectory "test-install" $ \ base -> do
+                  let damlPath = DamlPath (base </> "daml")
+                      options = InstallOptions
+                          { iTargetM = Just (RawInstallTarget "source.tar.gz")
+                          , iActivate = ActivateInstall True
+                          , iQuiet = QuietInstall True
+                          , iForce = ForceInstall False
+                          , iSetPath = SetPath False
+                          }
+
+                  setCurrentDirectory base
```

Comment:
Should this not be `withCurrentDirectory`? Any use of `setCurrentDirectory` freaks me out.